### PR TITLE
Change title of website

### PR DESCRIPTION
**Pull request by wd15**
Address #510

Change title of website to PFHub.

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/chimad-phase-field/557)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,7 @@ alt="Travis CI">
 
 ## Overview
 
-The CHiMaD phase field site is a location to compare and constrast
-phase field codes. See the
+PFHubis a location to compare and constrast phase field codes. See the
```
Comment (tkphd) [inline README.md]: "PFHub is"

**Comment by dschwen** [comment]
Did anyone actually Google "pfhub" before the decision was made to rename?

**Comment by drjamesawarren** [comment]
Did you? I’m afraid to now

Jim

Dr. James A Warren
Director, Materials Genome Program
Material Measurement Laboratory
National Institute of Standards and Technology
(301)-975-5708
________________________________
On: 15 November 2017 23:27, "Daniel Schwen" <notifications@github.com<mailto:notifications@github.com>> wrote:

Did anyone actually Google "pfhub" before the decision was made to rename?

**Comment by stvdwtt** [comment]
@dschwen Hmm...that's less than great. On one hand, it is so different that no one will get us mixed up, but on the other hand, we don't really want any association with a spammy news site.

Honestly, I could go either way in terms of sticking with PFHub or finding a new name.

**Comment by wd15** [comment]
This is why we have a review process!

**Comment by wd15** [comment]
I'll make this change permanent if I don't hear any dissenting views by Wednesday.

**Comment by wd15** [comment]
@tkphd, why don't we just use "Phase Field Community Hub" for now without the short version until that's approved and leave the repository at chimad-phase-field as an intermediary step?

**Comment by tkphd** [comment]
Unless there's a pressing need to change it today, I suggest we keep everything as is until legal gets back to me about PFhub, which should be within a week if not sooner.

**Comment by tkphd** [comment]
NIST legal needs 2 weeks to research and formulate a response. We should have an answer by December 15.

**Comment by tkphd** [comment]
Lo, we have an answer from Henry Wixon. Condensed version is as follows:
- We *can* use PFhub, but should seriously consider something else.
- It is improbable that our use would cause a "likelihood of confusion" in the minds of [pfhub.com][pfhub] consumers -- the primary legal test -- since scientific simulation software and personal finance are essentially unrelated.
  - Determination of a "likelihood of confusion" depends on specific details. Risky.
  - We are advised to seek a formal analysis from the DOC [General Law Division][gld] if we press ahead with PFhub.
- If Go Time Media *objects* to our use and *contends* that we're creating confusion, we would be forced to switch names.
- Given the technical nature of [chimad-phase-field][chimad], and the personal finance focus of [pfhub][pfhub], both of our uses of the mark *should* be permitted to co-exist.
  - Even so, if Go Time Media strongly objects to our existence, we would be required to change our name (again).
- The USPTO database only deals with registration applications. The simple fact that [pfhub][pfhub] uses the :tm: symbol means that they own the trademark. It is *not* expired.
- Counsel's personal opinion is that "pfhub" is no better than "chimad-phase-field" or "phase-field-hub".

My opinion? The legal team's worst-case scenario is that we have to change names, if [pfhub][pfhub] becomes aware of our existence and cares to object. Let's go ahead with **PFhub: The Phase Field Community Hub** at https://pages.nist.gov/pfhub and, if we have to change later on, fall back to https://pages.nist.gov/phase-field-hub.

[chimad]: https://pages.nist.gov/chimad-phase-field
[gld]: https://ogc.commerce.gov/collection/general-law-division
[pfhub]: http://www.pfhub.com

**Comment by tkphd** [comment]
Reviewed 4 of 5 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[_config.yml, line 22 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNIsuuJ69QHo_zqUB:-L0LNIsvnfCTayCpNrn3:b-26pkbi) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/_config.yml#L22)):*
> ```YAML
>   simmeta: https://github.com/usnistgov/chimad-phase-field/blob/master/_data/simulations
> 
> title: "PFHub: Phase Field Community Hub"
> ```

Please separate into 
```
title: PFHub
subtitle: Phase Field Community Hub
```

---

*[INVOLVEMENT.md, line 51 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNYRKlmRblWEII42l:-L0LNYRKlmRblWEII42m:b-j60ui) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/INVOLVEMENT.md#L51)):*
> ```Markdown
> <h4> Workshops </h4>
> 
> An integral component of PFHub is regular [workshops]({{ site.baseurl
> ```

`{{ site.title }}`

---

*[README.md, line 33 at r1](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-Kz1-1UvGydcW-2K8o6c-r1-33:-L0LNnopUw5L5_YSx3xP:b-j60ui) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/366761e4c37f1cd4d09e58771982dfd1e1fc6f33/README.md#L33)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

"PFHub is"
</blockquote></details>

`{{ site.title }}`

---

*[README.md, line 9 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNeSJ6dZA1kzz4t4u:-L0LNeSJ6dZA1kzz4t4v:ba0uzfx) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/README.md#L9)):*
> ```Markdown
> 
> <h1> <p align="center"><sup><strong>
> PFHub: Phase Field Community Hub
> ```

Can we use `{{ site.title }}: {{ site.subtitle }}` here?

---

*[workshops.md, line 12 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNsb7XEnOjAnr71XQ:-L0LNsb7XEnOjAnr71XR:b-ryayni) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/workshops.md#L12)):*
> ```Markdown
> ---
> 
> An integral component of [PFHub]({{ site.baseurl }}/#benchmarks) is
> ```

`[{{ site.title }}]`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557)*

**Comment by tkphd** [comment]
Merge conflicts with `master`.

**Comment by wd15** [comment]
Review status: 1 of 7 files reviewed at latest revision, 5 unresolved discussions.

---

*[_config.yml, line 22 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNIsuuJ69QHo_zqUB:-L0LgRD7p1VQ_H22GEOh:b-896fix) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/_config.yml#L22)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

Please separate into 
```
title: PFHub
subtitle: Phase Field Community Hub
```
</blockquote></details>

Done.

---

*[INVOLVEMENT.md, line 51 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNYRKlmRblWEII42l:-L0LgVuMlZ1J-mOCU-bd:b-896fix) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/INVOLVEMENT.md#L51)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

`{{ site.title }}`
</blockquote></details>

Done.

---

*[README.md, line 33 at r1](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-Kz1-1UvGydcW-2K8o6c-r1-33:-L0LgmxYxFDyWOVgjcxh:b4pq3wf) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/366761e4c37f1cd4d09e58771982dfd1e1fc6f33/README.md#L33)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

`{{ site.title }}`
</blockquote></details>

Likewise, not processed by Jekyll

---

*[README.md, line 9 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNeSJ6dZA1kzz4t4u:-L0LgXNosiSe0KC8sfoO:b-m5zzv7) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/README.md#L9)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

Can we use `{{ site.title }}: {{ site.subtitle }}` here?
</blockquote></details>

README.md is not used on the website, but it meant to be viewed on the GitHub project site and, consequently, is not processed by Jekyll.

---

*[workshops.md, line 12 at r2](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557#-L0LNsb7XEnOjAnr71XQ:-L0Lgr1xmJ2-PBv0Kna-:b-896fix) ([raw file](https://github.com/usnistgov/chimad-phase-field/blob/b45a734c6d4185939ea3c0ba182b25df7092ec80/workshops.md#L12)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

`[{{ site.title }}]`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557)*

**Comment by tkphd** [comment]
Reviewed 5 of 5 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/usnistgov/chimad-phase-field/557)*

**Comment by tkphd** [comment]
@amjokisaari, @drjamesawarren, @dschwen, @guyer, @stvdwtt:
The development version of the website is up and running at http://random-cat-557.surge.sh/. Please take a look and, if it's acceptable, give your approval through the "Review" button. Thanks!

**Comment by tkphd** [comment]
@dschwen you did not [vote](https://doodle.com/poll/shsqhizprgdnq7v4). Why?

**Comment by dschwen** [comment]
@tkphd I was not very invested in the naming choice after my entry got completely ignored and missed that Doodle. But the name is not a huge issue.

**Comment by tkphd** [comment]
OK, we'll have to advertise doodle polls more aggressively next time.

**Comment by drjamesawarren** [comment]
What awesome name did you suggest?

________________________________
On: 14 December 2017 18:55, "Daniel Schwen" <notifications@github.com<mailto:notifications@github.com>> wrote:

@tkphd<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftkphd&data=02%7C01%7Cjames.warren%40nist.gov%7C37c219e2f4a041ea76dc08d5434e2800%7C2ab5d82fd8fa4797a93e054655c61dec%7C1%7C0%7C636488925308833651&sdata=HhDIGPsxtvVXSB446RIhypmGvwWXG9o2T4Ae6lI1KvQ%3D&reserved=0> I was not very invested in the naming choice after my entry got completely ignored and missed that Doodle. But the name is not a huge issue.

**Comment by dschwen** [comment]
I did not suggest any "awesome" name, I offered the very straight forward and self explanatory domain http://phasefield.net

**Comment by tkphd** [comment]
OK, but that website already exists and is populated with your own thoughts about phase field. And, @wd15 can correct me if I'm wrong, but there's pushback from our overlords against moving off of the nist.gov domain.

**Comment by dschwen** [comment]
The name to go with that would obviously be something like "The Phase Field Network" (insert "Community" if necessary).

**Comment by dschwen** [comment]
The site has been - so far - only a placeholder. I actually only registered it during the last CHiMaD workshop.

**Comment by dschwen** [comment]
Yeah, your _overlords_ are sure being killjoys. Having the front page of the sit on an unrelated domain in a subdirectory is not ideal.

**Comment by tkphd** [comment]
Look, I'm sorry, but as far as I can tell from the [chat meeting](https://gitter.im/chimad-phase-field/chat-meeting-3), your proposal of phasefield.net never reached critical mass. It was briefly tossed around, then the mob moved on and circled in on pfhub. If you feel very strongly that this is a misstep, create an agenda item for the next chat meeting and we can revisit. Until then, I suggest we move forward on migration to the current consensus name and URL.

**Comment by dschwen** [comment]
There was not even tossing around (unless you count the comment "@dschwen too bad we don't know who owns that"). PFHub does seem to have solid support, so that is fine (doesn't mean I have to love it though).

**Comment by tkphd** [comment]
OK. Thank you for the feedback -- it is appreciated.

**Comment by wd15** [comment]
@dschwen: Sorry! I completely missed your suggestion and it didn't get added to the Etherpad so it didn't end up on the list of voting choices. I would certainly have found it acceptable. Anyway, maybe we should move on at this point though as we have other challenging issues with the website and anything is better than "CHiMaD Phase Field", which is gibberish to most people.

**Pull request merged**